packages/mosaic-tanstack-table-core/src/log-formatter.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Include all this code in the `logger.ts` file. There's no need for this to live in a separate file.